### PR TITLE
Update helm documentation

### PR DESCRIPTION
**Pull request by michaeldeift**

### CODE REVIEW

**Comment by michaeldeift** [comment]
CU-860pufe01

File: docs/Learn/Helm.md [hunk 1]
```diff
@@ -68,14 +67,27 @@ Will perform the following command:
...
+<img src="./img/helm-repo-add.png" width="450">
+
+
 - Navigate to the HELM under the Resources section in Komodor   
 - Switch to the Repositories tab  
 - Click “Add Repository”  
 - Specify the repository details - Name, URL, and Clusters to associate this repository with  
 - Save the repository  
 
+Test connection: Currently we are checking that the URL is accessible from a publicfrom public network only.
```
Comment (amit9192) [inline docs/Learn/Helm.md:89]: @michaeldeift this statement is a little unclear - "from a publicfrom public network only"

I would probably state it differently:
When adding a repo, Komodor verifies that the Repo is accessible from a public network.

Also - what happens if it's not accessible or it's a private repo?

File: docs/Learn/Helm.md [hunk 2]
```diff
@@ -68,14 +67,27 @@ Will perform the following command:
...
 
+In order to view and manage your repositories you need the following permissions:
+`manage:helm` (allowing the following: update, add and remove repos)
+`read:helm-repo`
+
 Adding a repository: 
 
+Adding a repository will add the repository to komodor agent that is running on the chosen clusters.
+
+<img src="./img/helm-repo-add-header.png" width="450">
```
Comment (amit9192) [inline docs/Learn/Helm.md:78]: Wouldn't it make more sense to add the images inline with the instructions below?

File: docs/Learn/Helm.md (hunk 1 above)
Comment (amit9192) [inline docs/Learn/Helm.md:89]: @michaeldeift this statement is a little unclear - "from a publicfrom public network only"

I would probably state it differently:
When adding a repo, Komodor verifies that the Repo is accessible from a public network.

Also - what happens if it's not accessible or it's a private repo?

**Review by amit9192 — COMMENTED** [review]
Otherwise looks good

File: docs/Learn/Helm.md (hunk 1 above)
Comment (michaeldeift) [reply docs/Learn/Helm.md:89]: In case it fails we display an error, but still allow adding it

File: docs/Learn/Helm.md (hunk 2 above)
Comment (michaeldeift) [reply docs/Learn/Helm.md:78]: @amit9192  can you show in the google doc?

Comment (amit9192) [reply docs/Learn/Helm.md:78]: Moved the images there

**Review by ChenKubaniKomodor — APPROVED** [review]

**Review by amit9192 — APPROVED** [review]

**Pull request merged**